support-lambdas/acquisitions-firehose-transformer/src/test/scala/com/gu/acquisitionFirehoseTransformer/LambdaSpec.scala

```diff
@@ -0,0 +1,109 @@
+package com.gu.acquisitionFirehoseTransformer
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.flatspec.AnyFlatSpec
+
+import com.gu.support.acquisitions
+import com.amazonaws.services.lambda.runtime.events.KinesisFirehoseEvent
+
+import io.circe.syntax._
+import org.joda.time.DateTime
+import com.gu.i18n.{Country, Currency}
+import com.gu.support.zuora.api.ReaderType
+import java.nio.ByteBuffer
+import collection.JavaConverters._
+import com.gu.acquisitionsValueCalculatorClient.model.AcquisitionModel
+import scala.concurrent.ExecutionContext
+
+
+class LambdaSpec extends AnyFlatSpec with Matchers {
```

Comment:
💯 shall we delete the old test file now? I think the new tests cover serialization just as well

Reply:
yeah makes sense - I'll remove that now! 👍